### PR TITLE
2099 Add Functional Use Category records to search index and results

### PR DESCRIPTION
**Pull request by mwfrost**
Closes: #2099 

To test this on your dev environment, first run `python manage.py search_index --rebuild` to refresh the index. Then search for terms that appear in the functional use category titles and descriptions.

### CODE REVIEW

File: dashboard/tests/functional/test_search.py [hunk 1]
```diff
@@ -384,11 +385,32 @@ def test_search_tag(self):
...
+        #
+        cat = FunctionalUseCategory.objects.filter(title="solvent").first()
+        cat.description = "A solvent is used to keep solutes dissolved in solution."
+        cat.save()
+
+        term = "dissolved"
+        qs = self._get_query_str(term)
+        response = self.client.get("/search/functionalusecategory/" + qs)
+
+        self.assertContains(response, '<em>dissolved</em>')
```
Comment (hausman-gdit) [inline dashboard/tests/functional/test_search.py:412]: `"` not `'`

Comment (mwfrost) [reply dashboard/tests/functional/test_search.py]: new commit just pushed

Comment (hausman-gdit) [reply dashboard/tests/functional/test_search.py]: thanks I see it, running through the steps now, nice to see this after the other search stuff add

**Review by hausman-gdit — APPROVED** [review]
Functions as expected,

**Pull request merged**